### ISSUE TITLE
numbers in tags

### ISSUE THREAD

**Issue by pihentagy** [sachingooo/anki-filtered-deck-by-tag#7]
I have created some decks from tags.

The tags also represent order in [this deck](https://ankiweb.net/shared/info/247695077), but when I create a filtered deck, the number disappears from the beginning.

**Comment by pihentagy** [comment]
Ah found it: https://github.com/sachingooo/anki-filtered-deck-by-tag/blob/main/anki-filtered-deck-by-tag/__init__.py#L92

I will hack it for myself, but don't know how others think about cleaning vs not cleaning the numbers…

**Comment by sachingooo** [comment]
Yeah, admittedly this is because I was using the AnKing deck for Step 1 and didn't like the numbers at the beginning. I'll include the config option to remove numbers in the next version

**Comment by pihentagy** [comment]
Cool, thanks
